flask_wiki/api.py

```diff
@@ -388,16 +397,23 @@ def languages(self):
 
     def search(self, term, ignore_case=True, attrs=['title', 'tags', 'body']):
         pages = self.index()
-        if not term:
-            return pages
-        regex = re.compile(term, re.IGNORECASE if ignore_case else 0)
         matched = []
-        for page in pages:
-            for attr in attrs:
-                if regex.search(getattr(page, attr)):
+        if term == "*" or term == "?":
+            return pages
+        elif not term:
+            for page in pages:
```

Comment:
This can be replace by a filter: https://docs.python.org/3.6/library/functions.html#filter or a comprehension list.

Reply:
J'ai finalement opté pour la solution 1 car je n'ai pas réussi à faire fonctionner la 2. On aurait dit que la fonction avait un problème avec `page.language` et retournait tout.